test/helpers/api-unit.helper.js

```diff
@@ -1,10 +1,27 @@
 import '../../website/src/libs/api-v3/i18n';
+import mongoose from 'mongoose';
 import { defaultsDeep as defaults } from 'lodash';
 import { model as User } from '../../website/src/models/user';
 import { model as Group } from '../../website/src/models/group';
 
-afterEach(() => {
+mongoose.Promise = require('q').Promise;
+
+mongoose.connect('mongodb://localhost/habitica-unit-tests');
```

Comment:
I've added this to the unit helper so we can actually write unit tests for mongoose operations if we'd like.